### ISSUE TITLE
Hexadecimal Display crashes Visual Studio

### ISSUE THREAD

**Issue by SamuelHenderson** [Microsoft/VSLinux#205]
Please use the following bug reporting template to help produce actionable and reproducible issues:

* A brief description
When trying to view variables in the  'Autos' and 'Locals' panels as hexadecimal Visual Studio becomes unresponsive and crashes.

* Expected results
The values of variables in the 'Autos' and 'Locals' panels are displayed as hexadecimal values.

* Actual results (with build output window contents if applicable)
Visual Studio becomes unresponsive.  Eventually a message is displayed asking if I would like to relaunch Visual Studio.

* VC++ version, Linux system name and version, GCC/GDB/gdbserver version, CPU arch, etc. If you are using a specific version of Linux on an embedded board, we might need to know about it to get a repro on similar hardware

    - Visual Studio 2017 Enterprise ver 15.4.1
    - Visual C++ For Linux ver 1.0.8
    - Fedora 26 (x86_64)
    - GNU gdb (GDB) Fedora 8.0.1-26.fc26
    - GNU gdbserver (GDB) Fedora 8.0.1-26.fc26

* Steps required to reproduce the error

1. Create a new Visual C++ For Linux project (Console, or Empty Project)
2. Add some source code like the following:
```
#include <cstdio>

int main()
{
    int i = 100;
    int j = 10000;
    printf("hello from ConsoleAppTest!\n");
    return 0;
}
```
3. Place a breakpoint
4. Start debugging application
5. In 'Autos' or 'Locals' window right click.
6. Select 'Hexadecimal Display' in context menu
7. Visual Studio will hang.

* Required packages and commands to install
```
$sudo dnf install gcc gcc-c++ gdb gdb-gdbserver
```
See [our contributing instructions](/CONTRIBUTING.md) for assistance.

**Comment by paulmaybee** [comment]
Bug is already opened against the MIEngine. https://github.com/Microsoft/MIEngine/issues/69. I will crosslink this post and raise its priority with the MIEngine.

**Comment by paulmaybee** [comment]
Fix is available in VS 15.8.
